client/src/App.css

```diff
@@ -355,13 +355,21 @@ button {
 }
 
 .myCarousel-item {
```

Comment:
Could you replace line 357 - 372 with the following code snippet? So the picture will not be stretched when window size is greater than 1000px.
```

.myCarousel {
  justify-content: center;
  background-color: lavenderblush;
}

.myCarousel-item {
  display: block;
  margin: auto;
  width: auto;
  height: auto;
  max-height: 400px;
}

```


 Also, add a className to Landing.js file on line 10 (as shown below). 
```

 <Container>
      <Row className="myCarousel">
          <MyCarousel />
      </Row>
```